### PR TITLE
Update tests for review page

### PR DESCRIPTION
**Pull request by morkune**
Please check my new tests! Thank you. ;)

### CODE REVIEW

File: cypress/integration/yelpcamp_tests/reviews.spec.js [hunk 1]
```diff
@@ -4,9 +4,52 @@ describe("Review feature tests", () => {
...
+  });
+
+  it("check review page", () => {
     cy.get(".card:first > p >").click();
     cy.get(".reviews-link").click();
     cy.get(".checked");
   });
+
+  it("editing a campground review", () => {
+    cy.get(".card:first > p >").click();
```
Comment (JuliusMorkunas) [inline cypress/integration/yelpcamp_tests/reviews.spec.js]: Is trailing `>` in a selector necessary?

**Comment by JuliusMorkunas** [comment]
I'd recommend moving all the tests one directory up. Is nesting everything to the `yelpcamp_tests` folder needed?

File: cypress/integration/sign_in.spec.js [hunk 2]
```diff
@@ -0,0 +1 @@
+/// <reference types="Cypress" />
```
Comment (JuliusMorkunas) [inline cypress/integration/sign_in.spec.js:1]: Do you really need this empty spec file? 🙂

Comment (morkune) [reply cypress/integration/sign_in.spec.js:1]: Maybe it is not good to leave this file empty for now, but it will be updated soon. :)

Comment (JuliusMorkunas) [reply cypress/integration/sign_in.spec.js:1]: I guess the file can be easily added as soon as it's needed.
I'd use camelCase for file naming by the way. 🙂

**Review by JuliusMorkunas — APPROVED** [review]

**Pull request merged**